### DISCUSSION TITLE
Possibility of Service Contracts to meet Corporate Policy requirements?

### DISCUSSION THREAD

**Discussion by codecae** [DaxStudio/DaxStudio#733] [Q&A] [answered]
Our company would enjoy the use of this software immensely, but our policy states that we must have a service agreement for either DAX Studio directly or an agreement for another product that includes DAX Studio as a component.

Is there someone who our company can reach out to to discuss topics such as these?

**Comment by dgosbell** [comment, marked as answer]
Sorry for the delay in responding, I've been doing some research on what would be involved with this, but unfortunately this is not something we are setup to be able to offer at the moment.

**Reply by codecae** [reply]
Thank you for your response!  I'll forward it on to the necessary teams.  If there's anything I can do to further this discussion, please feel free let me know!
